sdk/maps/azure-maps-search/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+## 2.0.0b1 (Unreleased)
```

Comment:
Can you please add a complete changlog describing all the API changes between this and the previous release.